src/watchmaker/__init__.py

```diff
@@ -278,8 +278,8 @@ def _get_config(self):
             data = watchmaker.utils.urlopen_retry(self.config_path).read()
         except (ValueError, urllib.error.URLError):
             msg = (
-                'Could not read the config from {0}! Please make sure your '
-                'config is available.'.format(self.config_path)
+                'Could not read the config file from the provided value {0}! Please '
```

Comment:
I think if we put quotes around the value, it will help indicate what watchmaker sees.

```suggestion
                'Could not read the config file from the provided value "{0}"! Please '
```

Reply:
Fixed